frontend/public/components/operator-lifecycle-manager/descriptors/status/index.tsx

```diff
@@ -79,9 +80,9 @@ export const StatusDescriptor: React.SFC<DescriptorProps> = (props) => {
     </div>
     <dd className="olm-descriptor__value">
       {
-        descriptor.displayName === 'Status'
-          ? (<StatusIconAndText status={value} iconName={value === 'Running' ? 'ok' : undefined} />)
-          : (<Capability descriptor={descriptor} capability={capability} value={value} namespace={namespace} />)
+        descriptor.displayName === 'Status' ?
```

Comment:
Nit: I think the convention is to put `?` and `:` at the beginning of their respective expressions

```tsx
descriptor.displayName === 'Status'
  ? (value === 'Running' ? <SuccessStatus title={value} /> : <Status status={value} />)
  : <Capability descriptor={descriptor} capability={capability} value={value} namespace={namespace} />
```

Reply:
Done